server/integration_tests/test_data/e2e_india_demo/howdoesliteracyratecomparetopovertyinindia/chart_config.json

```diff
@@ -4,60 +4,35 @@
     "categories": [
       {
         "blocks": [
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "scatterTileSpec": {
-                      "highlightTopRight": true
-                    },
-                    "statVarKey": [
-                      "Count_Person_Literate_scatter",
-                      "Count_Person_BelowPovertyLevelInThePast12Months_scatter"
-                    ],
-                    "title": "Population: Literate (${yDate}) Vs. Population Below Poverty Line (${xDate}) in Administrative Area 1 Places of India",
-                    "type": "SCATTER"
-                  }
```

Comment:
@pradh debug why [How does literacy rate compare to poverty in India?] doesn't show scatter on autopush now